### DISCUSSION TITLE
Newer linux distros are shipped with PipeWire instead of PulseAudio

### DISCUSSION THREAD

**Discussion by cjdg** [rustdesk/rustdesk#4767] [General]
I wanted to install it, but it needs pulseaudio, which breaks modern distros, which are shipped with PipeWire, and is compatible with pulseaudio, is there any plan to change the deb to stop asking for pulseaudio?

**Comment by b3uman** [comment]
Same here expected to use pipewire by defualt as it's the norm

**Reply by basilgello** [reply]
The change should be tribial: pulseaudio => `pulseaudio | pipewire-pulse`
